### PR TITLE
Drag drop marker

### PR DESCRIPTION
**Pull request by KnollFrank**
Making items draggable as mentioned in https://groups.google.com/g/mapsforge-dev/c/L_d1ZKXYlkI

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -4,7 +4,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.3'
+        classpath 'com.android.tools.build:gradle:4.2.1'
```
Comment (devemux86) [inline build.gradle:7]: _This is is unrelated to this pull request._

We cannot change Android Gradle plugin,
as IDEA is not compatible yet.

**Comment by devemux86** [comment]
This is a feature more related to specific app requirements.

Library's classes must remain unmodified to not break compatibility with user apps.
You can extend the needed classes and implement dragging as your app needs it.

There is already a bad implemented cluster feature,
let's not load the library with so much custom code.

I had implemented dragging externally in the past,
without changing VTM and breaking compatibility.

**Comment by devemux86** [comment]
If you are interested, we could include your custom implementation as a sample in [vtm-android-example](https://github.com/mapsforge/vtm/tree/master/vtm-android-example).

For the same reason, I moved the cluster experiment and its extended classes in the [cluster](https://github.com/mapsforge/vtm/tree/master/vtm-android-example/src/org/oscim/android/cluster) subfolder.
(see #858)

Similarly, you could create a "drag" subfolder with the extended classes for your custom implementation with a sample.

**Comment by KnollFrank** [comment]
Ok, I will try it, but I will take me a week or so....

**Pull request closed without merging**